### PR TITLE
Add route to change rate limit + set default config

### PR DESCRIPTION
**Pull request by petersg83**
- Add route to change rate limit
- Set rate limit to false by default
- Set autoOpen to false by default
- Fix test setup for API tests

### CODE REVIEW

**Review by ronronscelestes — APPROVED** [review]
bioutiful ❤️

**Review by jhoward1994 — APPROVED** [review]
🚀

**Review by gu-stav — COMMENTED** [review]
Nice one!

Does the config.set approach work for any setting you have in the BE?

**Comment by petersg83** [comment]
@gu-stav Not all config
All config that is read dynamically on each request : yes
If it's read only at startup and not read again after : no

**Comment by petersg83** [comment]
I add you @joshuaellis as a reviewer as I needed to change some e2e setup so the API tests pass

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "test:e2e": "yarn test:e2e:setup && playwright test",
     "test:e2e:codegen": "playwright codegen",
     "test:e2e:debug": "yarn test:e2e:setup && PWDEBUG=1 playwright test",
-    "test:e2e:setup": "yarn test:generate-app --appPath test-apps/playwright",
+    "test:e2e:setup": "yarn test:generate-app --appPath test-apps/playwright --template==../e2e/app-template",
```
Comment (joshuaellis) [inline package.json:52]:
```suggestion
    "test:e2e:setup": "yarn test:generate-app --appPath test-apps/playwright --template=../e2e/app-template",
```

**Review by joshuaellis — CHANGES_REQUESTED** [review]
small typo but otherwise thanks for doing this :)

File: package.json (hunk 1 above)
Comment (petersg83) [reply package.json:52]: good eye :)

**Comment by petersg83** [comment]
Something is broken in the setup, working on it

**Comment by petersg83** [comment]
should be fixed now

**Review by joshuaellis — APPROVED** [review]

**Review by gu-stav — APPROVED** [review]

**Pull request merged**